GitDiffMargin/Core/MarginCore.cs

```diff
@@ -329,6 +236,60 @@ public void FocusTextView()
             TextView.VisualElement.Focus();
         }
 
+        private void CheckBeginInvokeOnUi(Action action)
+        {
+            if (TextView.VisualElement.Dispatcher.CheckAccess())
+                action();
+            else
+                TextView.VisualElement.Dispatcher.BeginInvoke(action);
+        }
+
+        private void HandleFormatMappingChanged(object sender, FormatItemsEventArgs e)
+        {
+            if (e.ChangedItems.Contains(DiffFormatNames.Addition)
+                || e.ChangedItems.Contains(DiffFormatNames.Modification)
+                || e.ChangedItems.Contains(DiffFormatNames.Removed))
+                UpdateBrushes();
```

Comment:
📝 missing braces for multi-line condition. This situation is similar to the one that led to [SA1519](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1519.md), and IMO it was a mistake to omit this from the design of that rule (see DotNetAnalyzers/StyleCopAnalyzers#1488).